packages/discord.js/src/client/actions/ThreadListSync.js

```diff
@@ -29,7 +29,7 @@ class ThreadListSyncAction extends Action {
 
     for (const rawMember of Object.values(data.members)) {
       // Discord sends the thread id as id in this object
-      const thread = client.channels.cache.get(rawMember.id);
+      const thread = client.channels.cache.get(rawMember.user.id);
```

Comment:
This is also wrong. Don’t just add .user.id behind everything that has the name member. This is a threadMember from the API and the id is the id of the ThreadChannel, nothing to do with GuildMember or User.